numpy/lib/twodim_base.py

```diff
@@ -136,12 +204,11 @@ def flipud(m):
     return m[::-1, ...]
 
 
-def rot90(m, k=1):
+def rot90(m, k=1, axes=(0,1)):
     """
-    Rotate an array by 90 degrees in the counter-clockwise direction.
+    Rotate an array by 90 degrees in the plane specified by axes.
 
-    The first two dimensions are rotated; therefore, the array must be at
-    least 2-D.
+    Rotation direction is from the second towards the first axis.
 
```

Comment:
One last nitpick: I think this is backwards. In the original `rot90`, rotating `0` -> `1` was counter clockwise, meaning that we visualize the array as a matrix, with axis `0` pointing *down* and axis `1` to the right. In that case we rotate the first axis to the second and not vice versa (the axis `1` goes to minus axis `0`).